webui/react/src/services/api-ts-sdk/api.ts

```diff
@@ -12840,15 +12820,15 @@ export const InternalApiFetchParamCreator = function (configuration?: Configurat
         /**
          * 
          * @summary Get a list of jobs in queue.
-         * @param {number} [paginationOffset] The number of records to skip before returning results.
-         * @param {number} [paginationLimit] The amount of records limited in the results.
+         * @param {number} [offset] Pagination offset.
```

Comment:
nit: the previous comment seems more exact as to how it's implemented. keep it?

Reply:
it seems redundant to me to explain what "pagination offset" is. it's standard limit-offset pagination, if it doesn't work like one and needs a verbose explanation then we're doing something wrong.